index.html

```diff
@@ -499,6 +498,30 @@ <h3>Scalability modes</h3>
         </tr>
       </tbody>
     </table>
+    <section id="addingmodes*">
+      <h3>Guidelines for addition of <code>scalabilityMode></code> values</h3>
+      <p>When proposing a <code>scalabilityMode</code> value, the following principles should be followed:</p>
+        <ol>
+          <li>
+            The proposed <code>scalabilityMode</code> MUST define entries to the table in
+            Section 6, including values for the Scalabilty Mode Identifier, spatial and
+            temporal layers, Resolution Ratio and Inter-layer dependency.
+          </li>
+          <li>
+            The Scalability Mode Identifier SHOULD be consistent with the existing naming
+            scheme in which "LxTy" denotes a <code>scalabilityMode</code> with <code>x</code>
+            spatial layers and <code>y</code> temporal layers, "SxTy" denotes a
+            <code>scalabilityMode</code> with <code>x</code> simulcast streams, each of which
+            has <code>y</code> temporal layers, and post-pending an "h" denotes a resolution
+            ratio of 1.5 to 1.
```

Comment:
Where in the naming structure should additional information (like "_KEY") be put? "Post-pending" can argue for either L2T2_KEY or L2T2h_KEY

The advice should probably also say that if the mode is defined by an external specification, that specification's naming pattern should be followed unless it would cause collision. (I'm waiting for HEVC to show up with a completely different pattern...)

Aside: I think the mixing of upper and lower case with _KEY and h looks ugly, but guess that has been decided over in AV1-land.

Reply:
Here is some suggested text:

"            The Scalability Mode Identifier SHOULD be consistent with naming schemes used
            in existing specifications such as [[AV1]] Section 6.7.5. The AV1 naming scheme
            utilizes  "LxTy" to denote a <code>scalabilityMode</code> with <code>x</code>
            spatial layers using a 2:1 resolution ratio and <code>y</code> temporal layers.
            "LxTyh" denotes <code>x</code> spatial layers with a 1.5:1 resolution ratio and
            <code>y</code> temporal layers. "SxTy" denotes a <code>scalabilityMode</code>
            with <code>x</code> simulcast encodings with a 2:1 resolution ratio, with each
            simulcast encoding containing <code>y</code> temporal layers. "SxTyh" denotes
            a 1.5:1 resolution ratio. In addition, [[AV1]] Section 6.7.5 defines "LxTy_KEY"
            and "LxTy_KEY_SHIFT" modes."

Does this address your comment?

Reply:
Assuming that the proposed text is OK.